controller/Controller.php

```diff
@@ -69,7 +69,7 @@ public function setLanguageProperties($lang)
      * As a side effect, set the HTTP Vary header if a choice was made based on
      * the Accept header.
      * @param array $choices possible MIME types as strings
-     * @param string $accept HTTP Accept header value
+     * @param string $accept HTTP Accept header value
```

Comment:
Hmm, doesn't show on the GitHub UI. But we have a NBSP (unicode U+00A0), not a space here (U+0020).